recipes/artery-font-format/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+file(COPY example.arfont DESTINATION ${CMAKE_BINARY_DIR})
```

Comment:
```suggestion
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```

Reply:
I couldn't apply the suggestion for github so I did it in manually in fb66db6364fa4a991032ed6cf6f83b95db447ec9.

Reply:
Great.
I made these suggestions from github web.
So I don't guarantee these are 100% correct.